src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -154,42 +154,42 @@ def process_item(item):
 
 ### JavaScript
 
-To access our tracing features, you will need to install our Tracing integration:
+To access our tracing features, you will need to install our Tracing package `@sentry/apm`:
 
 ```bash
 $ npm install @sentry/browser
```

Comment:
Isn't this line unnecessary? `@sentry/apm` depends on `@sentry/browser`.

Reply:
Nope we need it